### PR TITLE
Coding standards fixes

### PR DESCRIPTION
**Pull request by spacedmonkey**
As I wish to use this project in a VIP GO site, I have taken it apon myself to fix the coding standards and security issues with this plugin. 

I run this against VIP GO 2.0.0 and Core 2.0.0 standards. Many of the fixes are automated, but other I did by hand. 

I also improved some of the comments (mainly the commons defining in the plugin). 

One other thing, many of the filters and actions were not named correctly. Instead of removing them, I have added new filters and  deprecated the old ones.

### CODE REVIEW

**Comment by spacedmonkey** [comment]
I also had to fix travis to that test run :(

Remove the comment rules for now, as that is a much larger issue, that should be managed here.

Looping in @rmccue @tfrommen in hopes of a review.

**Comment by kadamwhite** [comment]
Thanks for taking on this work!

I would like to ask you to pull this apart into two pull requests, one for coding standards changes and one for filter name changes. This will mean that some additional changes will be needed to disable CS sniffs in comments if they apply to the older filter syntax, but this is a lot to review in one go and the filter name change is much more impactful and will be surprising for existing users of the plugin; we’ll be able to move much faster on the individual pieces if they’re split out. 

Thanks!

**Comment by spacedmonkey** [comment]
Thanks for the feedback @kadamwhite . 

I have broken up this PR into 2. There is #59 for the filter names. 

I know it is a a lot of code, but you can see the linters passing in travis now, meaning at least the code is valid.

File: inc/admin/profile/namespace.php [hunk 1]
```diff
@@ -57,24 +63,29 @@ function render_profile_section( WP_User $user ) {
...
+		<p>
+			<a href="<?php echo esc_url( $personal_url ); ?>"><?php esc_html_e( 'Create personal access token', 'oauth2' ); ?></a>
+		</p>
 	<?php endif ?>
 	<?php
 }
 
 /**
- * Render a single row.
+ *  Render a single row.
```
Comment (kadamwhite) [inline inc/admin/profile/namespace.php]: How is this correct? This is not aligned with anything else in the comment.

File: inc/admin/profile/namespace.php [hunk 2]
```diff
@@ -147,11 +158,11 @@ function render_token_row( WP_User $user, Access_Token $token ) {
 	?>
 	<tr>
 		<td>
-			<p><?php echo $name ?></p>
-			<p><?php echo implode( ' | ', $details ) ?></p>
+			<p><?php echo $name; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></p>
+			<p><?php echo implode( ' | ', $details ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></p>
```
Comment (kadamwhite) [inline inc/admin/profile/namespace.php]: Is there a reason we don't just add `esc_html_e` here?

**Pull request merged**